docs/platform/howto/billing-assign-projects.rst

```diff
@@ -0,0 +1,24 @@
+Assign projects to billing groups
+=================================
+
+When you create a new project, you can assign it to a billing group, which lets you :doc:`use the same payment card and consolidate costs</docs/platform/concepts/billing-groups>`. 
+
+You can also assign the project to another billing group later in the `Aiven Console <https://console.aiven.io/>`_. 
+
+.. note::
+
+    Projects can only be assigned to one billing group. If a project is already assigned to one billing group and you assign it to another, it will be removed from the first group.
+
+To assign projects to a billing group:
+
+#. In the project, click **Billing**.
+
+#. Select the billing group you want to assign the project to.
+
+#. Click the **Projects** tab.
+
+#. Click **Assign projects**.
+
+#. Select the projects to assign to this billing group and click **Assign projects**.
+
+#. Click **Cancel** to close the dialog box. You will see the newly assigned projects listed in this billing group. 
```

Comment:
This comment applies to the current PR and the other one covering articles on creating and managing a billing group: How about adding final section "Related reading" (or similar) to all of the billing-related articles and cross-link between the three articles (for creating BG, managing BG, and assigning project to BG).